src/completion/acncompletionassist.cpp

```diff
@@ -41,7 +41,8 @@ std::unique_ptr<ProposalsBuilder> AcnCompletionAssistProcessor::createKeywordsPr
     return provider;
 }
 
-TextEditor::IAssistProcessor *AcnCompletionAssistProvider::createProcessor() const
+TextEditor::IAssistProcessor *AcnCompletionAssistProvider::createProcessor(const TextEditor::AssistInterface *assistInterface) const
```

Comment:
what is the purpose of the assistInterface? maybe we could use it somehow in the Processor?